Automattic/jetpack
# Backup plugin: Add the jetpack-identity-crisis package to composer.json

#### Changes proposed in this Pull Request:
* The Backup plugin uses the `jetpack-identity-crisis` package, so let's make sure that the `composer.json` file includes this dependency.
* The plugin currently includes the `jetpack-identity-crisis` package because the `jetpack-sync` package [requires](https://github.com/Automattic/jetpack/blob/master/projects/packages/sync/composer.json) it. However, it's good to document the plugin's dependencies directly in the plugin's `composer.json` file.

#### Jetpack product discussion
* n/a

#### Does this pull request change what data or activity we track or use?
* no

#### Testing instructions:
* Build the Backup plugin and make sure that the `vendor` directory contains `automattic/jetpack-identity-crisis`.
